lib/sms_api.rb

```diff
@@ -19,23 +19,24 @@ def authorization
   def sms_deliver(phone, code)
     return stubbed_response unless end_point_available?
 
-    response = client.call(:enviar_sms_simples, message: request(phone, code))
+    response = client.call(:enviar_sms, message: request(phone, code))
     success?(response)
   end
 
   def request(phone, code)
-    { autorizacion:  authorization,
-      destinatarios: { destinatario: phone },
-      texto_mensaje: "Clave para verificarte: #{code}. Gobierno Abierto",
-      solicita_notificacion: "All" }
+    { usuario: Rails.application.secrets.sms_username,
```

Comment:
```suggestion
    {
      usuario: Rails.application.secrets.sms_username,
```

Since the closing bracket is on a different line, how about applying the same criterion to the opening line?